outbound-dialer/src/main/java/org/motechproject/nms/outbounddialer/service/impl/TargetFileServiceImpl.java

```diff
@@ -130,6 +136,16 @@ private String targetFileName() {
     }
 
 
+    private void insertTargetFileAuditRecord(TargetFileNotification tfn, String status) {
```

Comment:
How about I remove the if altogether and pass `new TargetFileNotification()` instead of `null`?